TestON/drivers/common/cli/onosclidriver.py

```diff
@@ -933,7 +934,11 @@ def addHostIntent( self, hostIdOne, hostIdTwo ):
             else:
                 main.log.info( "Host intent installed between " +
                            str( hostIdOne ) + " and " + str( hostIdTwo ) )
-                return main.TRUE
+                match = re.search('id=0x([\da-f]+),', handle)
```

Comment:
This can be useful for the other intent types as well. In my tests I am now looking for the intent state of each intent I submit